docs/topics/telemetry.md

```diff
@@ -35,3 +44,21 @@ Telemetry can be routed to different application insights instance be specifying
   }
 }
 ```
+
+### Windows Specific Differences (wip)
+
+Currently, Windows CSE will only log telemetry to the aks-engine Application Insights instance, even
+if `applicationInsightsKey` is set. In a future change set, this feature will be enabled.
+
+### Collection Settings and Results
+
+aks-engine has the concept of a system defined aks-engine Application Insights key as well as a user
+defined key. The table below describes where telemetry is logged based on the configuration
+settings.
+
+| enableTelemetry  | applicationInsightsKey set | result |
+| ------------- | ------------- | ------------- |
+| true | true | both user and aks-e telemetry is tracked |
```

Comment:
i think we should stick with aks-engine instread of aks-e unless other docs refer to aks-e.
@CecileRobertMichon thoughts?

(same for below refs)

Reply:
`AKS Engine` is the preferred wording in docs

Reply:
Updated all references to be `AKS Engine`